### PR TITLE
[admin] More runtime info: podName and nodeIp

### PR DESCRIPTION
**Pull request by geropl**
Companion PR for: https://github.com/gitpod-com/gitpod/pull/5157

### CODE REVIEW

File: chart/templates/server-deployment.yaml [hunk 1]
```diff
@@ -188,6 +188,8 @@ spec:
               key: auth-providers.json
         - name: BRANDING_CONFIG
           value: {{ .Values.branding | toJson | quote }}
+        - name: ADMIN_LINKS
```
Comment (csweichel) [inline chart/templates/server-deployment.yaml:191]: AFAIK the plan was to make those links a "com" concern rather  than introduce generic means here.
What made you change your mind?

Comment (geropl) [reply chart/templates/server-deployment.yaml:191]: > What made you change your mind?

Nothing, I just followed the pattern here: Admin interfaces and extension points are implemented in `core`, actual implementation and usage in `com`.

Comment (geropl) [reply chart/templates/server-deployment.yaml:191]: Some details we can discuss and switch easily, e.g. move the dashboard part from `com` to `ee`, which would make total sense.

But breaking the existing pattern by introducing even more hinges is not worth the cost IMO. But let's discuss in sync.

Comment (csweichel) [reply chart/templates/server-deployment.yaml:191]: I wasn't asking to break patterns. One pattern that we have is that in `com` we can add additional properties, like those links.

I really don't think we need to configurable links here, but this is a com only concern.
We need the API extension in ws-manager to get the data - I don't think we need to add/maintain configurable links here.

**Review by csweichel — APPROVED** [review]
LGTM

**Comment by csweichel** [comment]
super nit: the two `[protocol] *` commits are really one `[ws-manager-api] *` change

**Pull request merged**